sensor_msgs/JointState[] goals
---
bool success
float64 distance
